index.js

```diff
@@ -100,20 +105,24 @@ async function demoHW() {
     // create cairo backed canvas
     const canvas = new Canvas(width, height);
     canvas.tagName = "CANVAS";
-
-    const surface = CanvasKit.MakeWebGLCanvasSurface(canvas);
```

Comment:
This is important change. Instead of drawing every frame to one surface I'm creating surface for rendering each frame.
This helps to understand is it skottie animation issue or rendering to surface